docs/guides/using-watt-with-node-config.md

```diff
@@ -0,0 +1,175 @@
+# Using Watt With Node Config
+
+[Node-config](https://www.npmjs.com/package/config) is a popular configuration management package that helps organize settings across different deployment environments in your application. It creates a unified configuration system that works seamlessly with both [Watt](https://platformatic.dev/docs/watt/overview) and other `npm` modules.
+When building a Watt application with multiple services, each service can maintain its own independent configuration using `node-config`. This allows different services to use different environment configurations as needed.
+
+
+
+## Installation and Setup
+
+First, install `node-config` in your Watt application:
+
+```sh
+npm install config
+```
+
+Create a `config` directory and set up your default configuration:
+
+```sh
+mkdir config
+touch config/default.json
+```
+
+In `config/default.json`, add your base configuration:
+
+```sh
+{
+  "foo": "bar"
+}
+```
```

Comment:
This section install config at the root, but it’s service specific now. Also the directory that it creates is  incorrect.